org.opent2t.onboarding.insteon/js/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "opent2t-onboarding-org-opent2t-onboarding-insteon",
```

Comment:
Nest and wink are the same way.

The way the opent2t library works, it starts with opent2t and then adds onboarding and then adds what's in the manifest, which is org.opent2t.onboarding.insteon. So you need the double level. 

I think this should be changed to what you suggested in the near future.